soap/src/java/com/zimbra/soap/mail/message/GetContactFrequencyRequest.java

```diff
@@ -13,12 +18,10 @@
     @XmlAttribute(name=MailConstants.A_EMAIL, required=true)
     private String contactEmail;
     /**
-     * @zm-api-field-description list of frequency graphs to return. time range values should be concatenated together
-     * Values are one or more of "d(day)w(week)m(month)".
-     * request can have any combination of values "dwm" like "dw", "wm", "mdw", etc.
+     * @zm-api-field-description
```

Comment:
I would have something in the description - perhaps "list of contact frequency graphs to return"
May be worth generating the docs under soap/ and checking that they look OK.

Reply:
I think I was going to put something there, and then forgot. good catch.